File: micro-service-eureka-service/src/main/java/demo/Eureka2ServerApp.java
package demo;

import org.springframework.boot.*;
import org.springframework.boot.autoconfigure.*;
import org.springframework.cloud.netflix.eureka.server.*;

import java.lang.invoke.*;

/**
 * Hello world!
 *
 * @author Chen
 */
@SpringBootApplication
@EnableEurekaServer
public class Eureka2ServerApp {
  public static void main(String[] args) {
    SpringApplication.run(MethodHandles.lookup().lookupClass(), args);
  }
}

File: micro-service-eureka-service/src/main/java/demo/Eureka2ServerApp.kt
package demo

import org.springframework.boot.*
import org.springframework.boot.autoconfigure.*
import org.springframework.cloud.netflix.eureka.server.*

/**
 * Hello world!
 *
 * @author Chen
 */
@SpringBootApplication
@EnableEurekaServer
class Eureka2ServerApp : CommandLineRunner {
  override fun run(vararg args: String?) {
    println("---------- application started")
  }
}

fun main(args: Array<String>) {


  SpringApplication.run(Eureka2ServerApp::class.java, *args)
}
